eng/pipelines/mirror-container-images.yml

```diff
@@ -0,0 +1,39 @@
+trigger: none
+pr: none
+
+parameters:
+  - name: Images
+    type: object
+    default:
+      - source: ghcr.io/chaos-mesh/chaos-daemon:v2.1.4
+        mirror: azsdkengsys.azurecr.io/mirror/chaos-mesh/chaos-daemon:v2.1.4
+      - source: ghcr.io/chaos-mesh/chaos-mesh:v2.1.4
+        mirror: azsdkengsys.azurecr.io/mirror/chaos-mesh/chaos-mesh:v2.1.4
+      - source: ghcr.io/chaos-mesh/chaos-dashboard:v2.1.4
+        mirror: azsdkengsys.azurecr.io/mirror/chaos-mesh/chaos-dashboard:v2.1.4
+      - source: ubuntu/squid
+        mirror: azsdkengsys.azurecr.io/mirror/ubuntu/squid
+
+jobs:
+  - job: MirrorImages
+    displayName: Mirror Container Images
+
+    pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
+      vmImage: MMSUbuntu20.04
+
+    steps:
+      - ${{ each image in parameters.Images }}:
+        - task: Docker@2
+          displayName: Login to ${{ image.mirror }} ACR
+          inputs:
+            command: login
+            containerRegistry: ${{ split(image.mirror, '.')[0] }}
```

Comment:
Does this work in the devops templating engine?

Reply:
Yep! Patrick tried it out on something else a few weeks ago which is how I discovered it.

Reply:
Or...wait it doesn't. But I guess login still worked. Let me iterate on it.

Reply:
I guess the login command seems to trim under the hood anyway, so I will change it to just pass `image.mirror`.

Reply:
Oh! No it does work, but I was logging the old value. Let me fix the display name log to also trim.